### PR TITLE
Added string and boolean definations

### PR DESCRIPTION
**Pull request by darahask**

### CODE REVIEW

File: LexicalAnalysis/lexer.l [hunk 1]
```diff
@@ -13,8 +13,10 @@ keywords (if|else|elif|loopif|show|take|return|void|start|integer|double|string|
 %%
 
 {keywords}                              {printf("Keyword: %s\n",yytext);}
-[0-9][0-9]*			         {printf("Integer: %s\n",yytext);}	
-[0-9]+"."[0-9]+			 {printf("Double: %s\n",yytext);}
+[0-9][0-9]*					            {printf("Integer: %s\n",yytext);}	
+[0-9]+"."[0-9]+				            {printf("Double: %s\n",yytext);}
+"true"|"false"                          {printf("Boolean: %s\n",yytext);}
```
Comment (shobhi1310) [inline LexicalAnalysis/lexer.l]: I don't think you can put it  like this, cause you will also have to make the compiler aware of what it exactly is in true or false

Comment (shobhi1310) [reply LexicalAnalysis/lexer.l]: separate it out.

**Review by shobhi1310 — CHANGES_REQUESTED** [review]

File: LexicalAnalysis/lexer.l (hunk 1 above)
Comment (darahask) [reply LexicalAnalysis/lexer.l]: so i should keep "true" as one and "false" as other?

Comment (darahask) [reply LexicalAnalysis/lexer.l]: @shobhi1310 just check it

Comment (shobhi1310) [reply LexicalAnalysis/lexer.l]: it is fine now

**Review by shobhi1310 — APPROVED** [review]

**Pull request merged**